cpp/examples/minimal_build/Dockerfile_static_system_libs

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM alpine:3.12
```

Comment:
Why do we need to use `alpine:3.12` image instead of `ubuntu:bionic` image for static system libs?
If we can use `ubuntu:bionic`, can we use `bash` in `run_static.sh`?

Reply:
Because alpine packages static versions of various dependencies. We would have to build them manually on ubuntu. This would add to the maintenance burden and increase the test duration.

Reply:
Ubuntu provides static libraries by `-dev` packages.
For example, `libzstd-dev` provides `libzstd.a`: https://packages.ubuntu.com/focal/amd64/libzstd-dev/filelist

Do we need more files than `.a`?

Reply:
I didn't know. Is it ok to update to `ubuntu:focal`? It has a package for thrift and `brotli-dev` contains `.a`s too.

Reply:
Yes. Ubuntu  Forcal is a new LTS.